src/day-1-toy/func.py

```diff
@@ -1,6 +1,17 @@
 # Write a function is_even that will return true if the passed in number is even.
 
+def is_even (num):
+	if int(num)%2==0:
+		return "true"
```

Comment:
This works, but consider using True (without quotes) instead of "true" (with quotes) here.